paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -181,6 +191,13 @@ public class PaparazziPlugin : Plugin<Project> {
         it.description = "Record golden images for variant '${variant.name}'"
       }
       recordVariants.configure { it.dependsOn(recordTaskProvider) }
+      val cleanRecordTaskProvider = project.tasks.register("cleanRecordPaparazzi$variantSlug") {
+        it.group = VERIFICATION_GROUP
+        it.description = "Clean and record golden images for variant '${variant.name}'"
+        it.dependsOn(deleteSnapshots, recordTaskProvider)
+        recordTaskProvider.get().mustRunAfter(deleteSnapshots)
```

Comment:
it's possible to set the task ordering from within the `cleanRecord` task itself, but unsure if we should?

Reply:
i'm uncertain as well, given that we have to eagerly get the record task in another task to do so.  Maybe let's just revert to the previous approach?